pkg/graph/schema.resolvers.go

```diff
@@ -199,6 +200,10 @@ func (r *queryResolver) Systems(ctx context.Context, after *string, first int) (
 	return conn, nil
 }
 
+func (r *queryResolver) TestDates(ctx context.Context, requestID uuid.UUID) ([]*models.TestDate, error) {
```

Comment:
I think @jim mentioned this on another PR, but I was hoping we could make this a query-able attribute of the accessibility request, rather than have to do a separate fetch for it.

Reply:
Fixed!